lib/GenomeFileUtil/core/GenbankToGenome.py

```diff
@@ -388,37 +566,35 @@ def _find_input_files(self, input_directory):
                 f"The input directory does not have any files with one of the "
                 f"following extensions {','.join(valid_extensions)}.")
 
-        logging.info(f"Found {len(genbank_files)} genbank files")
-
         input_files = []
         for genbank_file in genbank_files:
             input_files.append(os.path.join(input_directory, genbank_file))
 
         return input_files
 
     def _join_files_skip_empty_lines(self, input_files):
-            """ Applies strip to each line of each input file.
-            Args:
-                input_files: Paths to input files in Genbank format.
-            Returns:
-                Path to resulting file (currenly it's the same file as input).
-            """
-            if len(input_files) == 0:
-                raise ValueError("NO GENBANK FILE")
-            temp_dir = os.path.join(os.path.dirname(input_files[0]), "combined")
-            if not os.path.exists(temp_dir):
-                os.makedirs(temp_dir)
-            ret_file = os.path.join(temp_dir, os.path.basename(input_files[0]))
-
-            # take in Genbank file and remove all empty lines from it.
-            with open(ret_file, 'w', buffering=2 ** 20) as f_out:
-                for input_file in input_files:
-                    with open(input_file, 'r') as f_in:
-                        for line in f_in:
-                            line = line.rstrip('\r\n')
-                            if line.strip():
-                                f_out.write(line + '\n')
-            return ret_file
+        """ Applies strip to each line of each input file.
+        Args:
+            input_files: Paths to input files in Genbank format.
+        Returns:
+            Path to resulting file (currenly it's the same file as input).
+        """
+        if len(input_files) == 0:
+            raise ValueError("NO GENBANK FILE")
```

Comment:
This line will never get executed because of `if len(genbank_files) == 0:` logic in `_find_input_files` func

Reply:
so is safe to delete L582-583?

Reply:
Looks that way to me

Reply:
👍